src/pyOpenMS/tests/unittests/test_features.py

```
@@ -0,0 +1,428 @@
+from __future__ import print_function
+import pyopenms
+import copy
+import numpy as np
+import os
+import pandas as pd
+
+# Import shared test helpers from conftest.py
+from conftest import (report, _testMetaInfoInterface, _testUniqueIdInterface, _testParam, _testStrOutput)
+
+# Helper function for testFeatureMap
+def testDataProcessing(dp=pyopenms.DataProcessing()):
```

Ruff (0.14.10)

Do not perform function call `pyopenms.DataProcessing` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)